src/modules/client/components/ClientProfileCard.tsx

```diff
@@ -415,7 +415,8 @@ const ClientProfileCard: React.FC<Props> = ({ client }) => {
               <Stack sx={{ flexGrow: 1, maxWidth: '300px' }} gap={1} mt={2}>
                 <ClientPermissionsFilter
                   id={client.id}
-                  permissions='canEditClient'
+                  permissions={['canEditClient', 'canViewClientName']}
```

Comment:
lets add a comment here too about why `canViewClientName` is included

Reply:
Actually.. instead of scattering these around, we could also choose to just keep the change to `src/routes/protected.tsx` with a comment there @martha. This is just a safeguard not an expected scenario. Either way is fine with me, but if we have it in multiple places lets comment in all of them. thanks

Reply:
I was wondering about that too. I agree, since this is just a safeguard, I'll remove the checks to avoid unnecessary complexity and keep the safeguard in only 1 place